jetty-jmh/src/main/java/org/eclipse/jetty/util/thread/strategy/jmh/EWYKBenchmark.java

```diff
@@ -89,6 +92,14 @@ public static void setupServer() throws Exception
     @TearDown(Level.Trial)
     public static void stopServer() throws Exception
     {
+        try
+        {
+            IO.delete(directory);
```

Comment:
If we ever want to run these benchmarks on Windows, then this behavior (cleanup / delete at end) will not work (due to file system locks).
Delete on start is usually the better way.

Reply:
sure shouldn't be a problem but this means we will leave a bunch of files.

Reply:
we cannot really as this directory is created as a tmp file in setup method so we do not know the directory name..